code/datums/components/space_dive.dm

```diff
@@ -8,17 +8,24 @@
 	var/surface_time = 1 SECONDS
 	/// Traits added during phasing (and removed after)
 	var/static/phase_traits = list(TRAIT_MAGICALLY_PHASED, TRAIT_RUNECHAT_HIDDEN, TRAIT_WEATHER_IMMUNE)
+	/// Type of turf that is divable
+	var/diveable_turf = /turf/open/space
+	/// A decal we can dive from, and escape into (but only one enter)
+	var/diveable_decal = /obj/effect/decal/cleanable/vomit/nebula
```

Comment:
I do wonder a little if the presence of vars such as this indicate a component so specific that it has no particular purpose being modular... however the fact that these were all components made turning this from a species into a mob a breeze so I guess it's fine 😅